libs/sr-db/src/Database/DataManagers/LightConeData.ts

```diff
@@ -32,7 +32,10 @@ export class LightConeDataManager extends DataManager<
   constructor(database: SroDatabase) {
     super(database, storageKey)
     for (const key of this.database.storage.keys)
-      if (key.startsWith(storageHash) && !this.set(key, {}))
+      if (
+        key.startsWith(storageHash) &&
+        !this.set(key.split(storageHash)[1], {})
```

Comment:
feel free to add the opposite of `toStorageKey` ->`toDataKey`?

Reply:
dataKey is taken, so I used toDatumKey